### DISCUSSION TITLE
Move mongoose packages to mongoosejs namespace?

### DISCUSSION THREAD

**Discussion by hasezoey** [Automattic/mongoose#12137] [General]
This is a discussion / question about moving mongoose / mongodb related packages to the [mongoosejs](https://github.com/mongoosejs) namespace, the packages i mean for this discussion are:
- [mquery](https://github.com/aheckmann/mquery)
- [mpath](https://github.com/aheckmann/mpath)
- [kareem](https://github.com/vkarpov15/kareem)

Shouldn't these packages be moved to the mongoosejs namespace so that they can be better maintained (especially if the original author is not around anymore to maintain them), or is this blocked by something like "no author approval"?

**Comment by vkarpov15** [comment]
This would be great. @aheckmann do you have any thoughts on transferring mquery and mpath to mongoosejs org? Sorry for the ping out of the blue.

**Reply by aheckmann** [reply]
Good idea. Done

**Reply by vkarpov15** [reply]
Thanks!
